curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f407ea37ad6e181b90462e.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 65f407ea37ad6e181b90462e
+title: Step 14
+challengeType: 20
+dashedName: step-14
+---
+
+# --description--
+
+You could assign each `i` parameter to `self.i` as you did before. Although in this case you have few lines to repeat, one way to avoid this repetition is using the `super()` function. `super()` enables you to refer implicitly to the parent class: `super().__init__(x, y)` calls the `__init__` method of the parent class.
```

Comment:
```suggestion
You could assign each `i` parameter to `self.i` as you did before. Although in this case, you have few lines to repeat, one way to avoid this repetition is using the `super()` function. `super()` enables you to refer implicitly to the parent class: `super().__init__(x, y)` calls the `__init__` method of the parent class.
```